unitTests/utils/SearchboxResizeTest.ts

```diff
@@ -0,0 +1,81 @@
+import * as Mock from '../MockEnvironment';
+import { Searchbox } from '../../src/ui/Searchbox/Searchbox';
+import { ISearchboxOptions } from '../../src/ui/Searchbox/Searchbox';
+import { $$ } from '../../src/utils/Dom';
+import _ = require('underscore');
```

Comment:
same for forEach/each, you extract it from underscore
`import { forEach } from 'underscore';`